innovation.game.php

```diff
@@ -2833,7 +2840,12 @@ function getActivePlayerIdsInTurnOrderStartingWithCurrentPlayer() {
             $current_player_id = self::getCurrentPlayerUnderDogmaEffect();
         }
 
-        $current_player_no =  self::getUniqueValueFromDB(self::format("SELECT player_no FROM player WHERE player_id={current_player_id}", array('current_player_id' => $current_player_id)));
+        if ($current_player_id < 0) {
```

Comment:
what is this for?

Reply:
This is necessary because we re-added the special achievement check for Monument after each card transfer, and when it's the initial meld, it's technically everyone's turn so we don't have an active player. So we arbitrarily choose a player (in this situation no one would get a special achievement on an initial meld anyway, but maybe in debug mode, we'd trigger it by melding the right cards)